clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -167,3 +167,42 @@ func LatencyMapToPerfData(latency map[string]*LatencyMetric) *PerfData {
 	}
 	return perfData
 }
+
+type void struct{}
+
+var mem void
+
+type SvcBackends struct {
+	name string
+	lock sync.Mutex
+	// backends is a map: object svc_ip->svc_backend_ip->void
+	backends map[string]map[string]void
```

Comment:
backendsMapByIP?

Reply:
I guess that the comment was about cryptic variable name, not about data type used. What do you think about renaming this map to contain the information that is provided in comment?